samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake-json_serializable/lib/src/model/nullable_class.dart

```diff
@@ -122,7 +122,7 @@ class NullableClass {
   )
 
 
-  final List<Object>? arrayNullableProp;
```

Comment:
`Object` doesn't have to be a `Map`, it can also be `String`,`num`,`List` ,....

Reply:
I'm not sure where that change came from... it isn't mine. It came from a different commit. I didn't do anything to the `dart-dio` side of things.

Reply:
This change is needed because the "Object" type in Dart does not equate to the "object" type in the openApi spec. Rather, openApi generic objects are equivalent to Dart "Map" types and should be treated as such.

Reply:
then it would be list<map<string, object?>>

Reply:
In most cases Dart deserializes a JSON object to `Map<String, dynamic>` so it could be `List<Map<String, dynamic>>` to be specific, but `List<Map>` should be fine.